src/lazyframe.js

```diff
@@ -163,10 +165,10 @@ const Lazyframe = () => {
     if (params[1]) {
       params = params[1];
       const hasAutoplay = params.indexOf('autoplay') !== -1;
-      return hasAutoplay ? params : params + '&autoplay=1';
+      return hasAutoplay ? params : params + '&autoplay=1&rel=0';
```

Comment:
If the user wants to disabled related videos in the end that could be done by adding it to the src URL. What do you think?

Reply:
Yes, I needed this for my case as a default, because the users always seem to forget this. I think it should be the default, because you don't want recommended videos that could come from competitors on your website. So I am leaving this in my fork, but you are free to remove it.
What do you think about it?

Reply:
I hear you, but I think this is too opinionated. If this is added the user will not be able to override the setting. I will restore to original